### DISCUSSION TITLE
Who is using Hyperapp?

### DISCUSSION THREAD

**Discussion by aarongarciah** [jorgebucaran/hyperapp#1021] [Show and tell]
Would be nice to see a list of some real world websites built with Hyperapp. This would improve the trust in the project for newcomers that want to use Hyperapp to ship production-ready code.

Maybe a section like **Who is using Hyperapp?** in the README.md is enough.

What do you think?

**Comment by jorgebucaran** [comment]
@aarongarciah I think that's a good idea. We also have https://github.com/hyperapp/awesome-hyperapp.

**Comment by lukejacksonn** [comment]
Has anyone built any "real world websites" with the latest API? I have a few hyperapp apps in the wild but most of them use an older build as I haven't had the time to refactor them all and am now waiting till 2.0 to build any more.

I'm guessing qiita uses `> 1.0.0`  right @jorgebucaran ? That would be a good start.

**Comment by ThobyV** [comment]
Yes it's true most devs are waiting for the 2.0 API implementation to start making production apps everywhere.

**Comment by selfup** [comment]
I am currently making something but it is not deployed.

Holding off until 2.0 as well though. Just JSX changes for the most part, as core logic has been ironed out.

Great question though!

**Comment by loteoo** [comment]
https://hyperapp.rocks/

**Comment by aarongarciah** [comment]
@loteoo hyperapp.rocks looks great 😄

**Comment by selfup** [comment]
I am using it to send api calls to a go server for serial port communications.

Packaged as an executable (windows/mac/linux). Not a public facing website, but LAN facing 🎉 

https://github.com/selfup/hmrcmd

Example UI: https://user-images.githubusercontent.com/9837366/45041314-984cff80-b02d-11e8-97f7-1fbbeb2899d1.png

**Comment by joedski** [comment]
I'll grant that I haven't done anything practical with it... but at ~1kb, it seems to me like it'd make the perfect library for interactive widgets on static sites, usefulness as a general app framework aside.  It at least got me interested enough to play around with it a bit.  ([TodoMVC](https://github.com/joedski/musings/tree/master/Javascript/TodoMVC%20Tests/hyperapp), [A dirty impure attempt at async components](https://github.com/joedski/musings/tree/master/Javascript/Lazy-Loaded%20Views%20in%20Various%20View%20Libraries/hyperapp-v1))

I'll admit I am kinda also waiting for the official release of V2 :P

**Comment by 11111000000** [comment]
We use hyperapp v1 in production (injectable widget, ~3000 installations)

**Comment by ibudiallo** [comment]
We use it at Thankful.ai to build dynamic pages for end-users to provide information. We use v1 for our backend.

**Comment by minhthanh3145** [comment]
Just wanna share in case anyone is interested. I have recently used Hyperapp to create [a local web app that allows you to visualize your repository](https://github.com/minhthanh3145/Repository-Investigator). 

It's my first time using _Hyperapp_ along with _d3js_, had a lot of fun. I honestly thought this is a great framework, but It'd probably need more documentation. 

Personally (not in this but in a project I am currently working on) I had to read through the source code to know how to make a POST request to upload a file properly (boring, but necessary task). Cheers !

**Comment by minhthanh3145** [comment]
I built Another Writing Application, a writing workspace where you can gather sources, read, search for and write stuffs.

 https://github.com/minhthanh3145/Another-Writing-Application

Shout out in Hyperapp for being such a great framework.

**Comment by Jesspu** [comment]
So we have two "websites" that we use in production. They are really hybrid mobile apps that run on android tablets in the cab of big rigs. One of them uses hyperapp 1 (Workflow), and one uses the latest version of hyperapp 2 (TripScan). When we started building Workflow, hyperapp 2 was nowhere near ready, and I am sure at some point we will migrate it.

We actually won best in show at Freight Waves this year with our two apps. They use hyperapp, ionic capacitor, and some custom plugins to hook into Platform Sciences android ecosystem. We also use pubnub with a REST fallback, local store and forward, and Scanbot for document scanning (in TripScan). 

https://vimeo.com/415982372
